wlauto/instrumentation/trace_cmd/__init__.py

```diff
@@ -164,11 +164,12 @@ def on_run_init(self, context):
             raise InstrumentError('trace-cmd instrument cannot be used on an unrooted device.')
         if not self.no_install:
             host_file = context.resolver.get(Executable(self, self.device.abi, 'trace-cmd'))
-            self.trace_cmd = self.device.install_executable(host_file)
+            self.trace_cmd = self.device.install_if_needed(host_file)
```

Comment:
This should be just `install` rather than `install_if_needed` to be consistent with existing behavior.